PageObjects/systemConsole.py

```diff
@@ -2,16 +2,17 @@
 # Notifications tab
 notifications_tab="xpath=//span[contains(text(),'Notifications')]"
 suppress_alarm_button="xpath=//span[contains(text(),'Suppress Alarm')]"
-suppress_alarm_ok_button="xpath=//span[contains(text(),'OK')]"
-suppress_alarm_title="""xpath=//*[@id="messagebox-1001_header-title-textEl"]"""
-suppress_alarm_popup_message="""xpath=//*[@id="messagebox-1001-msg"]"""
-selected_suppress_alarm_popup_message="""xpath=//*[@id="messagebox-1001-msg"]"""
-suppress_alarm_no_button="xpath=//span[@id='button-1007-btnEl']"
 suppressed_alarms_button="xpath=//span[contains(text(),'Suppressed Alarms')]"
-suppressed_alarm_ok_button="xpath=//span[contains(text(),'OK')]"
-suppressed_alarms_title="""xpath=//*[@id="window-1404_header-title-textEl"]"""
+popup_title = """xpath=//*[@id="messagebox-1001_header-title-textEl"]"""
```

Comment:
can be used as popup_title = "xpath=//*[@id='messagebox-1001_header-title-textEl']"